src/pytest/test_gc.py

```diff
@@ -137,6 +138,7 @@ def testGCIntegrationWithRedisFork(env):
     env.expect('FT.ADD', 'idx', 'doc1', 1.0, 'FIELDS', 'title', 'hello world').ok()
     env.expect('bgsave').equal('Background saving started')
     env.cmd('FT.DEBUG', 'GC_FORCEINVOKE', 'idx')
+    time.sleep(1)
```

Comment:
Is there a better way? This seems prone to failure

Reply:
@mnunberg can not think of any other way, we need the rdb save to finish before restart it.

Reply:
Couldn't you use `os.stat` the rdb filename? - might need to add something in rltest, but I don't want any `sleep` in tests. Not only does this slow the tests down, it also makes the tests non-deterministic in case there is a CPU spike and things slow down for a bit.

Reply:
@mnunberg I added another commit, let me know what you think.

Reply:
Still need to delete the `sleep`

Reply:
Ohh missed it